spec/javascripts/pages/referenceWidget.spec.js

```diff
@@ -0,0 +1,92 @@
+describe('existingReferenceWidget', function() {
```

Comment:
again: feels a touch light on the interaction tests to my (admittedly biased!) taste. i don't see anything testing what happens when a user (1) selects an input returned by the type-ahead search and (2) clicks enter. is that because that's deemed to be (1) testing the library (type-ahead clicking acts like we expect) or (2) testing something trivial (clicking submit calls the function we expect to handle submit?)